src/integrations/koaApollo.test.ts

```diff
@@ -38,6 +49,67 @@ describe('koaApollo', () => {
      expect(() => (<any>apolloKoa)({}, 'x')).to.throw(
        'Apollo Server expects exactly one argument, got 2');
   });
+  it('uploads a single file using multipart/form-data', async () => {
+    const UploadedFileType = new GraphQLObjectType({
+      name: 'UploadedFile',
+      fields: {
+        filename: { type: GraphQLString },
+        mime: { type: GraphQLString },
+      },
+    });
+
+    const TestMutationSchema = new GraphQLSchema({
```

Comment:
Can you add this schema to the test schema in integrations.test.ts? That way we only have one test schema, which will make the test easier and shorter to read.